.changeset/tough-mangos-repair.md

```diff
@@ -0,0 +1,8 @@
+---
+'@microsoft/parcel-transformer-markdown-html': patch
+'@microsoft/atlas-integration': patch
+'@microsoft/atlas-site': patch
+'@microsoft/atlas-css': patch
+---
+
+Update dependencies
```

Comment:
Nit: in my PR we have a discussion about the tense we should write changesets in :D Let's keep continuous just these last times, and later we probably want to do it in past.

Reply:
I think present is better than continuous